wagtail/admin/templates/wagtailadmin/edit_handlers/tabbed_interface.html

```diff
@@ -1,7 +1,9 @@
 {% load wagtailadmin_tags %}
 <ul class="tab-nav merged" role="tablist">
     {% for child in self.children %}
-        <li id="tab-label-{{ child.heading|cautious_slugify }}" class="{{ child.classes|join:" " }} {% if forloop.first %}active{% endif %}" role="tab" aria-controls="tab-{{ child.heading|cautious_slugify }}"><a href="#tab-{{ child.heading|cautious_slugify }}" class="{% if forloop.first %}active{% endif %}">{{ child.heading }}</a></li>
+        <li class="{{ child.classes|join:" " }} {% if forloop.first %}active{% endif %}" role="tab" aria-controls="tab-{{ child.heading|cautious_slugify }}">
+            <h2><a href="#tab-{{ child.heading|cautious_slugify }}" class="{% if forloop.first %}active{% endif %}">{{ child.heading }}</a></h2>
```

Comment:
Is it appropriate to use a heading for a tab? This feels more like navigation than document outline, I’m not entirely sure what the best practices are for this.

Reply:
1) The h2s in the submenus, e.g. 'Settings' - I am not sure about removing the h2s for these, as to me those are clearly headings for the menu items below.  Perhaps what we should do is add an id to the h2, and then an `aria-labelledby` on the `ul` of the submenu. Like this example from https://www.w3.org/WAI/tutorials/menus/structure/:

```
<nav aria-labelledby="mainmenulabel">
	<h2 id="mainmenulabel" class="visuallyhidden">Main Menu</h2>
</nav>
```

2) Searches - I'll remove that one.

3) Tabs - I don't think it is usual practice to add a heading for a tab - that was one Beth had added that I didn't spot and I'll remove it again. We probably carefully need to review the tabs functionality and raise a separate ticket for it if it hasn't been done already, as they can be quite complicated to get fully accessible, with some pitfalls e.g. https://simplyaccessible.com/article/danger-aria-tabs/

Reply:
Actually... for searches that is a reasonable use of an h3 for a nav element - it just needs `id` and `aria-labelledby` adding.

Reply:
👌 good to know, I wasn't aware of those recommendations.